src/meroxa/resources.py

```diff
@@ -1,17 +1,21 @@
 import json
+from typing import Any
 
-from .types import CreateResourceParams
-from .types import MeroxaApiResponse
-from .types import UpdateResourceParams
-
+from .types import MeroxaApiResponse, EnvironmentIdentifier
 from .utils import ComplexEncoder, api_response
 
-BASE_PATH = "/v1/resources"
+RESOURCE_BASE_PATH = "/v1/resources"
 
 
 class Status(object):
-    def __init__(self, state: str, last_updated_at: str) -> None:
+    def __init__(
+        self,
+        state: str,
+        last_updated_at: str,
+        details: str = None,
+    ) -> None:
         self.state = state
+        self.details = details
         self.last_updated_at = last_updated_at
```

Comment:
```suggestion
        details: str = None,
    ) -> None:
        self.state = state
        self.details = details
```
last updated isn't in the status object usually... just state and details